engine/runtime/src/test/java/org/enso/runtime/test/DependenciesTest.java

```diff
@@ -0,0 +1,17 @@
+package org.enso.runtime.test;
+
+import static org.junit.Assert.fail;
+
+import org.junit.Test;
+
+public class DependenciesTest {
+  @Test
+  public void noAkkaDependencies() {
+    try {
+      Class.forName("akka.actor.ActorSystem$");
+      fail("akka.actor.ActorSystem should not be on the classpath");
```

Comment:
This test is a bit clumsy. If `akka.actor.ActorSystem` is not on the class-path, it only means that `runtime/test` does not have `akka` dependency, it does not check that there are no akka classes in `runtime.jar`. However, I still think that this test is useful. Moreover, I have manually checked that `runtime.jar` does not contain any akka classes.